apps/web/src/pages/TalentNominations/NominationGroup/components/NominatedForList.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable formatjs/enforce-plural-rules */
+// NOTE: Not really plural, I guess?
```

Comment:
They are not really plural. This probably should be two strings with a conditional wrapping it, no?

Reply:
Hmm.. that seems reasonable :thinking: 

I think we have this as [part of the un-ignoring the literal strings ](https://github.com/GCTC-NTGC/gc-digital-talent/issues/13583#:~:text=Counts%20in%20parentheses%20after%20strings%20Iterable%20(x))so maybe that could be a ticket where we unignore and swap it for a conditional rather than plural string?